quartz/styles/base.scss

```diff
@@ -261,7 +261,8 @@ thead {
 
   article > & > a {
     color: var(--dark);
```

Comment:
can we move this inside the `[role="anchor"]` specifier so we get colour on the links :) and while you're at it, might as well collapse the whole specified to `article > & > a[role="anchor"]`

Reply:
applied.